kubernetes/job/resource.yaml

```diff
@@ -68,7 +68,7 @@ spec:
         runAsGroup: <%NODE_GROUP_ID%>
         fsGroup: <%NODE_GROUP_ID%>
       containers:
-      - name: razeedeploy-job
+      - name: "razeedeploy-job{{{ UUID }}}"
```

Comment:
```suggestion
      - name: "razeedeploy-job-{{{ UUID }}}"
```

Reply:
i dont want a hanging `-` if UUID is undefined. it'll be up to the templator to prefix the UUID with a `-`